scopes/design/ui/tooltip/tippy.module.scss

```diff
@@ -12,11 +12,19 @@ $textColor: var(--bit-text-color, #fff);
 
     .tippy-content {
       padding: 0.85em;
-      white-space: nowrap;
+      &[data-breakline='false'] {
+        white-space: nowrap;
+      }
+      &[data-breakline='true'] {
+        white-space: pre-line;
+      }
     }
 
     .tippy-svg-arrow {
       fill: $bg;
     }
   }
 }
+.breakline {
+  word-break: break-all;
+}
```

Comment:
this is redundant, include it in the existing `&[data-breakline='true']`

Also note the comment at the top of the file:
```
// NOTE: this file should only be used to apply 'teambit' theme to tippy's default global css. use separate file for adding scoped css.
```